### PR TITLE
HTML/CSS & JavaScript capstone project - Conference page

### PR DESCRIPTION
**Pull request by uisendy**
### TechLive Summit page - HTML/CSS and Javascript Capstone Project
**In this capstone project an online website for a conference/Summit page was developed, and the following was obtained**

- A Good choice of Topic, TechLive Summit Hosted in Abuja Nigeria :- Simple and Abstract event
- Maintained Colors, Typography and Layout with the Original Design
- Building and Linking Multiple Pages - Home and About pages
- Practical Implementation of Media Query to handle responsive Design
- Dynamic Pages for the featured speaker section using Javascript object
- HTML/CSS and Javascript Best Practises
- GitHub Flow best practised
- No Linters Error
- Good Project Documentation with GitHub README.md file and Loom visual documentation. 

[View the Project Documentation here](https://www.loom.com/share/b46d9147b8024b09940373ad2bc95763)

### CODE REVIEW

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @uisendy, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 
- well-described video :+1: 
- well-designed code :+1: 

## Required Changes ♻️
- [ ] Here, On the about page **DESKTOP** version, the footer should be present with a dark background.
- your website
Screenshot from 2022-07-29 00-17-12
- Original website
Screenshot from 2022-07-29 00-16-59

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] There is also no need for a partner's section on the About page in the desktop version.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by uisendy** [comment]
Special Thanks to [Deepakdanger](https://github.com/Deepakdanger) for the Eye. I appreciate the review.

File: about.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+                  2021 <br />
+                  PORT HARCOURT
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+    <footer class="about-footer-section">
```
Comment (danmainah) [inline about.html:159]: - [ ] @uisendy great work implementing the changes, kindly ensure the footer background in desktop view is in black as previously requested.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+          <div class="event-logo">
+            <img
+              class="event-logo-img"
+              src="./asset/images/techLiveLogo.png"
+              alt="event-logo"
+            />
+          </div>
+        </div>
+      </section>
+      <section class="past-summit-section">
```
Comment (danmainah) [inline about.html:118]: - [ ] Kindly check on the design of this section the images need to have opacity in the , check the comparison below 

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image |  image

File: about.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.1/css/all.min.css"
+      integrity="sha512-KfkfwYDsLkIlwQp6LFnl8zNdLGxu9YAA1QvwINks4PhcElQSvqcyVLLD9aMhXd13uQjoXtEKNosOWaZqXgel0g=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style/style.css" />
+    <title>About: TECHLive</title>
+  </head>
+  <body>
+    <header class="nav-bar-section">
```
Comment (danmainah) [inline about.html:31]: - [ ] [optional] Kindly check on your navbar its missing the top bar containing the social media links language and logout.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @uisendy , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


 # Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (uisendy) [reply about.html:159]: Thanks for the review. That's covered now. Thanks

File: about.html (hunk 2 above)
Comment (uisendy) [reply about.html:118]: image
Kindly note that the effect is a hover effect and was implemented. Thanks for the comment

**Review by Sahar-AbdelSamad — APPROVED** [review]
## Status Approved 🎉 🥳 

Hi @uisendy , :wave::wave:

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
✔️ Great job implementing all the changes requested by the previous reviewers
✔️ Your Project looks awesome 😍 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**